packages/openai/src/responses/openai-responses-language-model.ts

```diff
@@ -1082,6 +1220,28 @@ function isResponseOutputItemAddedReasoningChunk(
   );
 }
 
+function mapOpenAIImageFormatToMediaType(format: string): string {
```

Comment:
I know we have `detectMediaType` but it doesn't look exported, and here we actually have a format, we're just trying to make it fully qualified